pages/agenda/2018.tsx

```diff
@@ -1,281 +1,264 @@
 import React, { Fragment } from 'react'
```

Comment:
Looks OK at a glance. Hard to look in detail because the diff isn't obvious. Suggest we test prod and test side-by-side to check all the details are correct?

Reply:
that's how I reviewed it, side-by-side, but it would be great to have another set of eyes go over it so we can be sure.